api/uploader/archive_test.go

```diff
@@ -37,7 +37,7 @@ func TestUploader_DownloadZipURL(t *testing.T) {
 	cldtest.UploadTestAsset(t, cldtest.PublicID2)
 
 	params := uploader.CreateArchiveParams{
-		Tags:           cldtest.Tags,
+		PublicIDs:      []string{cldtest.PublicID, cldtest.PublicID2},
```

Comment:
why was tags removed from the test?
Shouldn't this be a separate test?

Reply:
There is a pretty similar test below that also tests the tags. This test can be rewritten to test public ids